readme.md

```diff
@@ -12,13 +12,13 @@ This web app lets you fly with your spaceship inside of an area. Every remote pl
 
 ## Navigation
 
-- [Interaction](#Interaction)
-- [Express + sockets, which ports do I have to open?](#Express-+-sockets,-which-ports-do-I-have-to-open?)
-- [Expose when hosting on localhost](#Expose-when-hosting-on-localhost)
-- [Start the server!](#Start-the-server!)
-- [Socket](#Socket)
-- [Socket events used](#Socket-events-used)
-- [Todo](#Todo)
+- [Interaction](#interaction)
+- [Express + sockets, which ports do I have to open?](#express-+-sockets,-which-ports-do-i-have-to-open?)
```

Comment:
Just a small tweak.
All special characters should be removed when adding it in to the navigation. Included: + and , (except for the - character)